### ISSUE TITLE
Support specifying an initial scale when deploying an app

### ISSUE THREAD

**Issue by eladidan** [convox/rack#894]
Imagining that there should be some syntax to specify the desired scale when deploying an app.
Since scale is per process, then it would likely be specified in the `docker-compose.yml` as a `label`.

My use-case: 
We use Convox to deploy batch tasks that run on a cron-like schedule. When deploying the app for the first time, it will launch with a default scale of 1, which means that the scheduled-task (it is not long running) will run repeatably until I change the scale to 0. By setting the scale to 0 as part of the deployment, I would ensure that it only ever runs when the CloudWatch scheduled event is triggered.

**Comment by snario** [comment]
+1

This could also fit with #353, adding all scale properties to the config file.

An example label:

```yml
labels:
    - convox.scale.count=-1
    - convox.scale.memory=1024
    - convox.scale.cpu=1024
```

**Comment by ddollar** [comment]
Going to address in #353 adding CPU as well.

**Comment by eladidan** [comment]
@ddollar we should re-open this.
My initial issue was about supporting initial process scale in the compose file, not cpu and memory, and that was not addressed in #1254
